scripts/jenkins/jobs-ibs/openstack-mkcloud.yaml

```diff
@@ -0,0 +1,489 @@
+- job:
+    name: openstack-mkcloud
+    node: openstack-mkcloud
+    description: |
+     "!!! This job is automatically pushed from GIT to ci.suse.de !!!
+      REMEMBER: changes to this job will be overwritten after a few hours - make sure to create a PR for your changes
+      This job runs https://github.com/SUSE-Cloud/automation/blob/master/scripts/mkcloud results are 
+      analyzed by https://github.com/SUSE-Cloud/automation/blob/master/scripts/jenkins/mkcloudresultcheck.pl
+      on mkcloudresultcheck@mkcloudresultcheck.cloud.suse.de every 15 minutes"
+
+    logrotate:
+        numToKeep: 100
+        daysToKeep: 300
+
+    properties:
+      - authorization:
+          cloud:
+               - job-build
+               - job-cancel
+               - job-configure
+               - job-delete
+               - job-discover
+               - job-read
+               - job-status
+               - job-workspace
+               - run-delete
+               - run-update
+               - scm-tag
+          anonymous:
+               - job-read
+
+    parameters:
+     - string:
+        name: cloudsource
+        default: develcloud5
+        description: "Can be something like develcloud3 (uses Devel:Cloud:*, see TESTHEAD parameter for Staging),  susecloud3 (uses SUSE:*), GM3 (uses the GM ISO)"
+
+     - string:
+        name: mkcloudtarget
+        default: all
+        description:  Set mkcloudtarget to define the mkcloud command line parameters .
+     - choice:
+         name: libvirt_typ
+         choices:
+           - kvm
+           - xen
+           - hyperv   
+
+     - choice:
+         name: networkingplugin
+         choices:
+            - openvswitch
+            - linuxbridge
+
+     - choice:
+         name: networkingmode
+         choices:
+             - gre
+             - vlan
+             - vxlan
+     - choice:
+         name: storage_method
+         description: 'Select storage method: default (let mkcloud decide), none (enforce none)'
```

Comment:
I change to | and text move to next line.Will update all.Without quotes it fails.